File: beecrowd/python/ad-hoc/ex1542.py
while True:
    e = input().split()
    if e == ['0']:
        break
    
    q, d, p = map(int, e)
    resp = int(((p*d) / (p-q)) * q)
    
    print(f'{resp} paginas') if resp != 1 else print(f'{resp} pagina')

File: beecrowd/c++/ad-hoc/ex1542.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main(){
    double q, d, p;
    while (true){
        cin>>q;
        if (q==0){
            break;
        }
        cin>>d>>p;
        double x = (p*d) / (p-q);        
        int resp = x * q;
        if (resp != 1){
            cout<<resp<<" paginas"<<endl;
        } else{
            cout<<"1 pagina"<<endl;
        }
    }
        

    return 0;
}